### ISSUE TITLE
[Accessibility] [Contrast] inherent contrast in the text 5952 members

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#862]
**Describe the bug**
The contrast in the text is 3:1. It should be 4,5:1. The font-size is approximately 9pt. The suggested size for the contrast is 18pt or 14pt bold.

**To Reproduce**
1. Go to '(https://muda.cambiatus.io/join?redirect=%2Fdashboard)'
2. Run Colour Contrast Analyser (CCA)
3. Check contrast between font and background is 3:1 instead 4,5:1

**Expected behavior**
The contrast must be 4,5:1 or more

**Screenshots**

**Cambiatus Env you are using**
Put the URL of the page you are trying to use

**Desktop (please complete the following information):**
 - OS:Windows
 - Browser Chrome
 - Version 129.0.6668.101 (Versão oficial) 64 bits

**Additional context**

**Comment by ceciliabelem** [comment]
For mapping all defects statistics, I just updated the spreadsheet. Also, there is a graphic to help understand some kinds of bugs and how they can impact users.
[Muda's Report Spreadsheet](https://docs.google.com/spreadsheets/d/11-ld-vQhmtcJhRtPTiUF4YP4bajFFz6U/edit?usp=sharing&ouid=114917425951998070257&rtpof=true&sd=true)
